### DISCUSSION TITLE
usually the third Sunday of the month?

### DISCUSSION THREAD

**Discussion by pdurbin** [bostonopen/bostonopen.github.io#2] [General]
I was thinking about listing our group at https://bostonsoftwarecrafters.com/events and putting something like "third or fourth Sunday of the month."

Our first two meetings were the third Sunday. The next meeting will be the fourth Sunday but I only bumped it a week because people might want to attend LibrePlanet.

In practice, I've checked with Holmes if this or that Sunday works (we're skipping all of February due to travel). I'm just thinking it might be nice to have some predictability. Maybe it's "usually the third Sunday of the month." 😄 Nothing's perfect.

Thoughts on this are welcome, of course!

**Comment by pdurbin** [comment]
The next meeting (in April) will be the fifth Sunday.

For May we might need to do an earlier Sunday. Hmm, but Mother's Day is probably a bad idea. 😬 😅

**Comment by HarshKapadia2** [comment]
Creating a group on [meetup.com](https://meetup.com) would be nice, as a lot of people check that.

I usually find all my Tech events there and I know a lot of people who do that as well.

**Reply by pdurbin** [reply]
@HarshKapadia2 would you be able to start a new thread about Meetup? (From a quick look, it seems to cost [$16.49 per month]( https://help.meetup.com/hc/en-us/articles/360001620472-Organizer-subscription-pricing) and I'm not interested in paying that personally.) This thread is about establishing a regular meeting time (third Sunday or whatever). Thanks! 😄

**Comment by pdurbin** [comment]
I've been thinking...

It's hard to promise the third Sunday of the month (or whatever) if there are only a couple organizers and one or both might be out.

**Reply by HarshKapadia2** [reply]
Yeah true. You can set up a schedule and differ from it if you aren't available. People usually say along the lines of "We have an event on the last Saturday of the month, unless specified otherwise." That's completely acceptable if the number of times it differs is not that frequent.

**Reply by pdurbin** [reply]
Yeah, "unless specified otherwise" gives us a nice out. 😄
